weka_upgrade_checker/weka_upgrade_checker.py

```diff
@@ -779,7 +778,7 @@ def __init__(self, machine_json):
             )
         )
         unique_snap_layers = list(set(snap_layers))
-        snap_result = any(x < weka_buckets and x != 0 for x in unique_snap_layers)
+        snap_result = any(x != weka_buckets and x != 0 for x in unique_snap_layers)
```

Comment:
Hagai confirmed this logic in WEKAPP-427366, so I will be approving this change.